quickstart/azure/tutorial-azure-kubernetes-service.md

```diff
@@ -0,0 +1,266 @@
+
+    ```bash
+    $ pulumi update
+    Previewing update (dev):
+
+        Type                                         Name                Plan       Info
+    +   pulumi:pulumi:Stack                          aks-dev             create
+    +   ├─ azure:core:ResourceGroup                  rg                  create
+    +   ├─ azure:ad:Application                      aks-app             create
+    +   ├─ azure:network:VirtualNetwork              vnet                create
+    +   ├─ azure:containerservice:Registry           acr                 create
+    +   ├─ azure:ad:ServicePrincipal                 aks-sp              create
+    +   ├─ azure:network:Subnet                      subnet              create
+    +   ├─ azure:ad:ServicePrincipalPassword         aks-sp-pwd          create
+    +   ├─ azure:role:Assignment                     acr-permissions     create
+    +   ├─ azure:containerservice:KubernetesCluster  aks                 create
+    +   └─ azure:role:Assignment                     subnet-permissions  create
+    
+    Resources:
+        + 11 to create
+    ```
+
+4.  Now, proceed with the deployment. 
+
+    ```bash
+    Do you want to perform this update?
+    > yes
+      no
+      details
+
+        Type                                         Name                Status      Info
+    +   pulumi:pulumi:Stack                          aks-dev             created
+    +   ├─ azure:core:ResourceGroup                  rg                  created
+    +   ├─ azure:ad:Application                      aks-app             created
+    +   ├─ azure:network:VirtualNetwork              vnet                created
+    +   ├─ azure:containerservice:Registry           acr                 created
+    +   ├─ azure:ad:ServicePrincipal                 aks-sp              created
+    +   ├─ azure:ad:ServicePrincipalPassword         aks-sp-pwd          created
+    +   ├─ azure:role:Assignment                     acr-permissions     created
+    +   ├─ azure:network:Subnet                      subnet              created
+    +   ├─ azure:containerservice:KubernetesCluster  aks                 created
+    +   └─ azure:role:Assignment                     subnet-permissions  created
+
+    Outputs:
+        kubecfg: "omitted"
+
+    Resources:
+        + 11 created
+
+    Duration: 7m30s
+
+    Permalink: https://app.pulumi.com/4c74356b41/aks/dev/updates/10
+    ```
+
+    To see the full details of the deployment and the resources that are now part of the stack, open the update permalink in a browser.
+
+5.  To view the provisioned resources on the command line, run `pulumi stack`. You'll also see a [stack output](/reference/stack.html#output) corresponding to the kubernetes config file you can use to access the AKS cluster we created.  
```

Comment:
Let's omit this section, as its extraneous.

Reply:
section 5?

Reply:
Yes please :)

Reply:
done